<ui version="4.0" >
 <class>FindFile</class>
 <widget class="QDialog" name="FindFile" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find in Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Pattern:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="textFind" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkCase" >
       <property name="text" >
        <string>Case Sensitive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkWholeWords" >
       <property name="text" >
        <string>Whole words</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkRecursive" >
       <property name="text" >
        <string>Recursive </string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="1" >
      <widget class="QComboBox" name="comboFileTypes" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
       <item>
        <property name="text" >
         <string>*.cpp; *.h</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>*.ui; *.qrc; *.txt</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Files:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="comboFindIn" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QToolButton" name="chooseDirectoryButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Find in:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkReplace" >
     <property name="text" >
      <string>Replace found text</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Replace with:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="textReplace" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="findButton" >
         <property name="text" >
          <string>&amp;Find</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton" >
         <property name="text" >
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>FindFile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>313</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel" >
     <x>107</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
